src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/meta.yml

```diff
@@ -0,0 +1,59 @@
+admin_meta:
+    path: /
+    methods: GET
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\ShopParameters\Meta:index'
+        _legacy_controller: AdminMeta
+
+admin_meta_search:
+    path: /
+    methods: POST
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\ShopParameters\Meta:search'
+        _legacy_controller: AdminMeta
+
+admin_meta_list_create:
+    path: /create
+    methods: GET
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\ShopParameters\Meta:create'
+        _legacy_controller: AdminMeta
+
+admin_meta_list_edit:
+    path: /edit/{metaId}
```

Comment:
i think its better to follow route naming like:
`resources/{id}/action`, so its like `metas/{metaId}/edit`, wdyt @matks ?